ReconstructOrder/workflow/multiDimProcess.py

```diff
@@ -488,8 +472,88 @@ def process_sample_imgs(img_io: Union[mManagerReader, PolAcquReader]=None,
 
                 export_img(img_io, img_dict, separate_pos)
 
-            elapsed_time=time.time()-start_time
-            print('Finish processing and exporting (t=%3.2f min)' % elapsed_time)
+            if save_ret_Zavg:
+                plt.close("all")  # close all the figures from the last run
+                img_io.z_idx = z_list[0]
+                img_dict = {}
+                retard   = physical_data_Zavg.retard
+                azimuth  = physical_data_Zavg.azimuth
+                for channel in list(set(ret_Zavg_names) & set(img_io.output_chans)):
+                    if channel == 'RetardanceZavg':
+                        img = im_bit_convert(retard * config.plotting.retardance_scaling, bit=16)
```

Comment:
I think you can just do `if 'RetardanceZavg' in img_io.output_chans:` without the for loop.